Osiris/Hooks.h

```diff
@@ -35,6 +35,7 @@ class Hooks {
     HookType sound;
     HookType surface;
     HookType viewRender;
+    VmtSwap networkChannel;
```

Comment:
Mb better to use HookType, that's my opinion

Reply:
i do not know the inner workings of the network park of this 400ms code, i did not port it to osiris, so i cant do that easy.

I renember in the old PR's of this topic some dude who worked on this said he prefers Vmt swap, so ther emight be come issues moving it to HookType.

ill look into it maybe i can figure something out tho.